ns1/rest/zones.py

```diff
@@ -42,20 +45,52 @@ def import_file(
         )
 
     def create(self, zone, callback=None, errback=None, **kwargs):
-        body = self._buildBody(zone, **kwargs)
+        zone_name, body = self._buildBody(zone, **kwargs)
         return self._make_request(
             "PUT",
-            "%s/%s" % (self.ROOT, zone),
+            "%s/%s" % (self.ROOT, zone_name),
+            body=body,
+            callback=callback,
+            errback=errback,
+        )
```

Comment:
still thinking about this, on the fence but still not quite sold on alternative:

* this implementation preserves the idea of the argument as FQDN, which makes it cumbersome for "views" use, but it preserves that 'zone' field is required (whereas 'name' is not)

* if we choose to make the 'zone' argument be the zone name, then ... it does match with the other methods, 'zone' would only be required if needed ... but we lose a bit in terms of making required arguments required.

this applies to update (and in records.py)